src/app/components/common/FundingSummary/FundingSummary.tsx

```diff
@@ -69,7 +69,7 @@ const FundingSummary = (props: Props) => {
       [FundingSummeryColumns.FUNDING_SUMMARY]: (value: string | number) => payments.find(payment => payment.id === value)?.name,
       [FundingSummeryColumns.FEE]: (value: string | number) => `${paymentsValue(Number(value).toFixed(2))} SSV`,
       [FundingSummeryColumns.VALIDATORS]: () => validatorStore.validatorsCount,
-      [FundingSummeryColumns.SUBTOTAL]: (value: string | number) => `${validatorStore.isMultiSharesMode ? Number(Number(paymentsValue(value)) * validatorStore.validatorsCount).toFixed(2) : paymentsValue(value)} SSV`,
+      [FundingSummeryColumns.SUBTOTAL]: (value: string | number) => `${validatorStore.isMultiSharesMode ? formatNumberToUi(Number(Number(paymentsValue(value)) * validatorStore.validatorsCount).toFixed(2)) : paymentsValue(value)} SSV`,
```

Comment:
I see a lot of such conversions and it looks like we should think of a better approach, possibly stricter typings. Here there is Number(Number()) one of them can be removed.

Reply:
+

Reply:
Lets look at this together later. value here should be typed as number.